src/protocolgame.cpp

```diff
@@ -2390,6 +2404,9 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 	msg.addByte(0x00); // can change pvp framing option
 	msg.addByte(0x00); // expert mode button enabled
 
+	msg.addString("http://static.tibia.com/images/store/");
```

Comment:
You also make it harder for ppl wanting to replace the website store.
Use a config value here aswell, default it to
.... .com/store/
Or smth like that